docs/notebooks/Umbrella_Integration_PySAGES_HOOMD.md

```diff
@@ -0,0 +1,281 @@
+  kernelspec:
+    display_name: Python 3
+    name: python3
+---
+
+<!-- #region id="p49wJ0IjLAVD" -->
+
+# Setup of the environment
+<!-- #endregion -->
+
+<!-- #region id="pF-5oR_GMkuI" -->
+We download and install the environment of HOOMD-blue and OpenMM with their respective plugins.
+<!-- #endregion -->
+
+```bash id="nMThqa-DjVcb"
+
+BASE_URL="https://drive.google.com/u/0/uc?id=1hsKkKtdxZTVfHKgqVF6qV2e-4SShmhr7&export=download"
+wget -q --load-cookies /tmp/cookies.txt "$BASE_URL&confirm=$(wget -q --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate $BASE_URL -O- | sed -rn 's/.*confirm=(\w+).*/\1\n/p')" -O pysages-env.zip
+rm -rf /tmp/cookies.txt
+```
+
+```python colab={"base_uri": "https://localhost:8080/"} id="25H3kl03wzJe" outputId="55526734-bcea-4d1a-f1ae-de0b017126b7"
+%env PYSAGES_ENV=/env/pysages
+```
+
+```bash id="V6MZXhOJMz7P"
+
+mkdir -p $PYSAGES_ENV
+unzip -qquo pysages-env.zip -d $PYSAGES_ENV
+rm pysages-env.zip
+```
+
+```python id="JMO5fiRTxAWB"
+import os
+import sys
+
+ver = sys.version_info
+
+sys.path.append(os.environ["PYSAGES_ENV"] + "/lib/python" + str(ver.major) + "." + str(ver.minor) + "/site-packages/")
+```
+
+<!-- #region id="lf2KeHt5_eFv" -->
+## PySAGES
+
+The next step is to install PySAGES.
+First, we install the jaxlib version that matches the CUDA installation of this collab setup. See the JAX documentation [here](https://github.com/google/jax) for more details.
+<!-- #endregion -->
+
+```bash id="RUX1RAT3NF9s"
+
+pip install -q --upgrade pip &> /dev/null
+# Installs the wheel compatible with CUDA 11 and cuDNN 8.2 or newer.
+pip install -q --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html &> /dev/null
+```
+
+<!-- #region id="mx0IRythaTyG" -->
+We test the jax installation and check the versions.
+<!-- #endregion -->
+
+```python colab={"base_uri": "https://localhost:8080/"} id="Z4E914qBHbZS" outputId="94391314-23b5-4726-f34a-fd927a0d4da1"
+import jax
+import jaxlib
+print(jax.__version__)
+print(jaxlib.__version__)
+```
+
+<!-- #region id="vtAmA51IAYxn" -->
+Now we can finally install PySAGES. We clone the newest version from [here](https://github.com/SSAGESLabs/PySAGES) and build the remaining pure python dependencies and PySAGES itself.
+<!-- #endregion -->
+
+```bash id="rEsRX7GZNJ_R"
+
+rm -rf PySAGES
+git clone https://github.com/SSAGESLabs/PySAGES.git &> /dev/null
+cd PySAGES
+pip install -q . &> /dev/null
+```
+
+<!-- #region id="IXryBllMNiKM" -->
+We can see, that the particle positions are indeed centered around the constraints we set up earlier. Also, we see the shape of the histograms is very similar to the expected analytical prediction. We expect this since a liquid of soft particles is not that much different from an ideal gas.
+<!-- #endregion -->
+
+<!-- #region id="LjPjqVjSOzTL" -->
+# Umbrella integration
+
+In the previous section, we demonstrated how PySAGES can be used to run a single simulation with a biasing potential.
```

Comment:
"In the previous section" refers to the other tutorial. Maybe Change to "In the previous tutorial" and add link (or at least the name) to that tutorial?